server/src/structs.rs

```diff
@@ -17,7 +20,7 @@ use uuid::Uuid;
 /// Arrow Cargo. The operator supplies the assets to the network,
 /// expects to receive and operate cargo shipments, and is expected to
 /// derive revenue from the operation.
-#[derive(Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize)]
+#[derive(Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize, ToSchema, IntoParams)]
 pub struct Operator {
```

Comment:
I'm not sure if we want to call this a `operator` as it's a bit too generic. An `operator` can be a single person operating the vtols from distance for instance as well.
It can also be someone sitting in a support center, answering the phone.
In this case, you want to describe a group/ company that supplies specific assets. How about AssetSupplier instead?

Reply:
I'm not concerned about the ppl reading the code, I'm more concerned about the claiming of a word/ entity for something that could mean something else as well in our context.
I can see us having actual "Operators" as a user function (like we have pilots as well) in the future, which we can't use then as it's already been given to a company function that, in my opinion, matches more to what a "Supplier" does.

> I can rename it to AssetOwner but the tricky thing is some people might own assets but they are not interested in operate these assets.
That is exactly why I think Supplier fits better

Reply:
I think for actual labor who operator vehicles (not in a business sense, but in a logistics sense), we can call them by their actual job title (ie. Pilot, Remote Controller, who maybe implement a 'Labor' trait or something).

I would see us actually have a Supplier type when it comes to compliance modules. So, Suppliers are OEMs supplying assets to Operators, who'd then register and operate assets on our network. We might need both Suppliers and Operators present on some legal documents generated by the compliance module, so claiming Supplier here would make things harder later on.